<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoGigDialogBase</class>
 <widget class="QDialog" name="GeoGigDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoGig Sync</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>431</width>
     <height>211</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Sync</string>
    </attribute>
    <widget class="QLineEdit" name="txtMessage">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>331</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>66</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSync">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmail">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Clone</string>
    </attribute>
    <widget class="QLineEdit" name="txtRemote">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>271</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDir">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>271</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelRemote">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Remote Host:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRemote_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Directory: </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClone">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Clone</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>GeoGigDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>GeoGigDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
